tests/componenttests/client/tests/control/ApplicationStateChangeTest.cpp

```diff
@@ -60,20 +60,26 @@ class ApplicationStateChangeTest : public ClientComponentTest
  *   Expect that a session is created on the server.
  *   Check that the object returned is valid.
  *
- *  Step 4: Change state to INACTIVE
+ *  Step 4: Create a new web audio player session
+ *   Create an instance of WebAudioPlayer.
+ *   Expect that web audio api is called on the server
+ *   Check that the object returned is valid.
+ *   Check that web audio player has been added.
```

Comment:
Can remove 'Check that web audio player has been added', no need to test that here.